autoversioner/github/client.go

```diff
@@ -1,16 +1,18 @@
-package tagger
+package github
 
 import (
 	"bytes"
 	"context"
 	"fmt"
 	"io/ioutil"
+	"time"
 
 	"github.com/google/go-github/github"
 	"golang.org/x/oauth2"
 )
 
 const (
+	// TagObjectTypeCommit is commit type gitobject
```

Comment:
this comment and the variable name are unclear

Reply:
change the variable name; and added a reference to github doc. 
Let me know if it is still unclear